versioned_docs/version-3.0.0-beta/Edge-Network/Encryption.md

```diff
@@ -0,0 +1,29 @@
+---
+id: encryption
+title: Encryption
+sidebar_label: Encryption
+slug: /Edge-Network/encryption
+---
+
+Every Deployment on Spheron is served over an HTTPS connection. The SSL certificates for these unique URLs are automatically generated free of charge.
+
+Any HTTP requests to your Deployments are automatically forwarded to HTTPS using the 308 status code:
+
+_HTTP/1.1 308 Moved Permanently_
```

Comment:
Let's do it on our CDN. let's leave this on the doc for now since we will implement this in our edge network.
@prakarsh-spheron, leave it.
@DusanNikolic11, how much time will it take to add this to our system?